README.md

```diff
@@ -1,45 +1,79 @@
+--->
+<!---Link to docs with further instructions
+--->
 
+<!---Things you can do in the Fabric view once the extension is installed
+--->
+<!---Introduction to Fabric view
+--->
+## Create and connect to local_fabric runtime
+Coming in a later version. See [issue #10](https://github.ibm.com/IBM-Blockchain/fabric-vscode-extension/issues/10) 
+<!---Short explanation with code-blocks
+--->
+<!---Screenshot of UI/Video of prototype click-through 
+--->
+<!---Link to docs with further instructions
+--->
+## Connect to a specified (remote) Fabric runtime and discover the existing resources
+<!---Short explanation with code-blocks
+--->
+![Connect to a specificed(remote)Fabric runtime and discover the existing resources](https://github.com/simran-sohanpal/blockchain-vscode-extension/blob/readmeupdates/client/media/resources.png "Connect to the network and discover the existing resources")
+<!---Link to docs with further instructions
+--->
+## Install new chaincode
+<!---Short explanation with code-blocks
+--->
+![Install new chaincode](https://github.com/simran-sohanpal/blockchain-vscode-extension/blob/readmeupdates/client/media/install%20chaincode%201.png "Install new chaincode")
+![Install new chaincode](https://github.com/simran-sohanpal/blockchain-vscode-extension/blob/readmeupdates/client/media/install%20chaincode%202.png "Install new chaincode")
+<!---Link to docs with further instructions
+--->
+## Instantiate new chaincode
+<!---Short explanation with code-blocks
+--->
+![Instantiate new chaincode](https://github.com/simran-sohanpal/blockchain-vscode-extension/blob/readmeupdates/client/media/instantiate%20chaincode%201.png
+ "Instantiate new chaincode")
+![Instantiate new chaincode](https://github.com/simran-sohanpal/blockchain-vscode-extension/blob/readmeupdates/client/media/instantiate%20chaincode%202.png "Instantiate new chaincode")
+<!---Link to docs with further instructions
+--->
+## Test new chaincode
+<!---Short explanation with code-blocks
+--->
+<!---Screenshot of UI/Video of prototype click-through 
+--->
+Coming in a later version. See [issue #18](https://github.ibm.com/IBM-Blockchain/fabric-vscode-extension/issues/18)
+<!---Link to docs with further instructions
+--->
+## Commands
+The IBM Blockchain extension provides commands in the Command Palette for working with chaincode files:
+<!---Table of commands with columns: 'command' and 'description'
+--->
 ## License <a name="license"></a>
-The source code files are made available under the Apache License, Version 2.0 (Apache-2.0), located in the [LICENSE](LICENSE) file.
\ No newline at end of file
+Hyperledger Project source code files are made available under the Apache License, Version 2.0 (Apache-2.0), located in the [LICENSE](LICENSE) file. Hyperledger Project documentation files are made available under the Creative Commons Attribution 4.0 International License (CC-BY-4.0), available at http://creativecommons.org/licenses/by/4.0/.
```

Comment:
This needs to be as it was before we aren't doing this under hyperledger so don't want them mentioned